src/app/notification/notification.component.ts

```diff
@@ -46,7 +46,6 @@ export class NotificationComponent implements OnDestroy {
     private _active: boolean;
     private _warning: Notification;
     private _notifications: Array<Notification> = [];
-    private _warningTimer: NodeJS.Timer;
     private _subscriptions: Subscription[] = [];
     private _warningTimeout: number = 30 * 1000; // warning is shown for 30 seconds unless closed manually
```

Comment:
remove line 50  private _warningTimeout: number = 30 * 1000; // warning is shown for 30 seconds unless closed manually